src/controllers/accountviewcontroller.cpp

```diff
@@ -130,7 +130,15 @@ void AccountViewController::importFromCSV(std::string& path)
         }
         m_accountInfoChangedCallback();
     }
-    m_sendToastCallback(StringHelpers::format(ngettext("Imported %d transaction from CSV.", "Imported %d transactions from CSV.", imported), imported));
+
+    if (imported == -1)
```

Comment:
No need to create a double if statement when `if(imported > 0)` already exists.

I would make it:
```cpp
if(imported > 0)
{
    for(const std::pair<const unsigned int, Group>& pair : m_account.getGroups())
    {
        if(!m_mapFilters.contains(pair.second.getId()))
        {
            m_mapFilters.insert({ pair.second.getId(), true });
        }
    }
    m_accountInfoChangedCallback();
    m_sendToastCallback(StringHelpers::format(ngettext("Imported %d transaction from file.", "Imported %d transactions from file.", imported), imported));
}
else
{
    m_sendToastCallback(_("Unable to import information from the file. Unsupported file type.")); 
    //make sure string is translatable!
}
```